### ISSUE TITLE
Could not build Objective-C module 'React'

### ISSUE THREAD

**Issue by productdevbook** [lottie-react-native/lottie-react-native#724]
![Screen Shot 2021-02-01 at 15 18 12](https://user-images.githubusercontent.com/38668796/106457894-b4745580-64a0-11eb-8765-7b2a99d9a860.png)
### Description

[Description of the bug]
Could not build Objective-C module 'React'

### Steps to Reproduce

1. xcode build

### Versions

```
"dependencies": {
    "@react-native-async-storage/async-storage": "^1.13.4",
    "@react-native-community/cameraroll": "^4.0.2",
    "@react-native-community/masked-view": "^0.1.10",
    "@react-native-firebase/analytics": "^10.5.1",
    "@react-native-firebase/app": "^10.5.0",
    "@react-native-firebase/dynamic-links": "^10.5.1",
    "@react-native-firebase/messaging": "^10.5.1",
    "@react-navigation/bottom-tabs": "^5.11.7",
    "@react-navigation/native": "^5.9.2",
    "@react-navigation/stack": "^5.14.2",
    "@shopify/restyle": "^1.4.0",
    "formik": "^2.2.6",
    "lottie-ios": "3.1.8",
    "lottie-react-native": "^3.5.0",
    "react": "17.0.1",
    "react-native": "0.64.0-rc.2",
    "react-native-camera": "^3.42.0",
    "react-native-gesture-handler": "^1.9.0",
    "react-native-iphone-x-helper": "^1.3.1",
    "react-native-keyboard-aware-scroll-view": "^0.9.3",
    "react-native-purchases": "^4.0.0",
    "react-native-reanimated": "software-mansion/react-native-reanimated",
    "react-native-redash": "^16.0.8",
    "react-native-safe-area-context": "^3.1.9",
    "react-native-screens": "^2.17.0",
    "react-native-status-bar-height": "^2.6.0",
    "react-native-svg": "^12.1.0",
    "yup": "^0.32.8"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@babel/runtime": "^7.12.5",
    "@react-native-community/eslint-config": "^2.0.0",
    "@svgr/cli": "^5.5.0",
    "@types/jest": "^25.2.3",
    "@types/react-native": "^0.63.2",
    "@types/react-test-renderer": "^16.9.2",
    "@types/yup": "^0.29.11",
    "babel-jest": "^26.6.3",
    "babel-plugin-module-resolver": "^4.1.0",
    "eslint": "7.14.0",
    "jest": "^25.1.0",
    "metro-react-native-babel-preset": "^0.64.0",
    "react-native-codegen": "^0.0.6",
    "react-native-rename": "^2.7.0",
    "react-test-renderer": "17.0.1",
    "typescript": "^4.0.1"
  },
  "jest": {
    "preset": "react-native",
    "moduleFileExtensions": [
      "ts",
      "tsx",
      "js",
      "jsx",
      "json",
      "node"
    ]
  }
  ```

**Comment by mrousavy** [comment]
same error here :(

**Comment by mrousavy** [comment]
Nevermind, found it. To everyone having the same problem, apply this patch using patch-package: https://gist.github.com/mrousavy/517a18fdabdc048bba238132047cb4b0

**Comment by el-lsan** [comment]
Any hope for a fix ?

**Comment by mrousavy** [comment]
> Any hope for a fix ?

@el-lsan did you read the thread you commented on?

**Comment by mrousavy** [comment]
This is fixed by my PR https://github.com/lottie-react-native/lottie-react-native/pull/738 which should land in a release soon.

**Comment by el-lsan** [comment]
@mrousavy sorry, there was no linked PR to this issue so I didn't expected it.
Anyway thanks for the fix and info 🙌  
Cheers

**Comment by mMarcos208** [comment]
same issue.

```
"lottie-ios": "^4.3.4",
"lottie-react-native": "^6.4.1",
"react-native": "0.73.1",
```


Any solution?

**Comment by utiq** [comment]
> same issue.
> 
> ```
> "lottie-ios": "^4.3.4",
> "lottie-react-native": "^6.4.1",
> "react-native": "0.73.1",
> ```
> 
> Any solution?

Add this to your podfile

```
linkage = ENV['USE_FRAMEWORKS']
if linkage != nil
  Pod::UI.puts "Configuring Pod with #{linkage}ally linked Frameworks".green
  use_frameworks! :linkage => linkage.to_sym
  $RNFirebaseAsStaticFramework = true # In case you use Firebase
end
```
 and then install like this: `USE_FRAMEWORKS=static bundle exec pod install`

you might have to do `rm -rf Pods && Podfile.lock`

**Comment by Arthur-bl-dev** [comment]
Same Xcode Version 16.0 (16A242d)

**Comment by Arthur-bl-dev** [comment]
> Same issue. Xcode Version 16.0 (16A242d)

Fixed! Updating to version 7.0
